xLights/TimingPanel.cpp

```diff
@@ -481,11 +481,9 @@ wxString TimingPanel::GetTimingString()
     // Fade in
     if ("" != TextCtrl_Fadein->GetValue() && "0" != TextCtrl_Fadein->GetValue() && "0.0" != TextCtrl_Fadein->GetValue() && "0.00" != TextCtrl_Fadein->GetValue()) {
         s += "T_TEXTCTRL_Fadein=" + TextCtrl_Fadein->GetValue() + ",";
-        if (Choice_In_Transition_Type->GetSelection() > 0) {
```

Comment:
Why not just a GetStringSelection() != "Fade" ?    We shouldn't be putting default values into the xsq.

Reply:
Okay... I see.. that may work.. so now we don't put Fade.. we'll only write it for nonFade and then can remove the upconversion... nice, thank you